### DISCUSSION TITLE
ODO meter topic

### DISCUSSION THREAD

**Discussion by conradhagemans** [tillsteinbach/WeConnect-mqtt#156] [General]
Can someone tell me in which topic I can find the ODO meter?
I want to keep track how many KM I drive daily.

Thanks.

**Comment by tillsteinbach** [comment]
What car are you driving?

**Reply by conradhagemans** [reply]
ID.4

**Reply by tillsteinbach** [reply]
Do you already have SW 3.0 in your car?

**Reply by conradhagemans** [reply]
No still running on 2.4 version. Battery upgrade is done! So waiting for the OTA update.

**Reply by tillsteinbach** [reply]
Ah, then you have to wait. Odometer is coming with 3.0

**Reply by conradhagemans** [reply]
Thanks.
